datajunction-server/datajunction_server/sql/dag.py

```diff
@@ -30,7 +30,7 @@ def _node_output_options():
     """
     return [
         joinedload(Node.current).options(
-            joinedload(NodeRevision.columns).options(
```

Comment:
How did you even think of it to try?

Reply:
@agorajek I read some guidance that indicated `joinedload` was for many-to-one relationships and `selectinload` was for one-to-many or many-to-many relationships. In the case of NodeRevision <-> Columns, it was actually one-to-many (disguised as many-to-many because of the intermediate join table), so I figured it made sense to try out `selectinload`. And it turned out to be much faster.

Reply:
Thanks @shangyian . That makes sense.